app/assets/javascripts/factories/ScorerFactory.js

```diff
@@ -527,19 +558,20 @@
         }
       }
 
-      function maxScore(total, docs, bestDocs, options) {
+      function maxScore(total, docs, ratedDocs, bestDocs, options) {
         if (self.manualMaxScore) {
           return self.manualMaxScoreValue;
         }
 
-        return runCode(total, docs, bestDocs, 'max', options);
+        return runCode(total, docs, ratedDocs, bestDocs, 'max', options);
       }
 
-      function score(total, docs, bestDocs, options) {
-        var maxScore  = self.maxScore(total, docs, bestDocs, options);
+      function score(total, docs, ratedDocs, bestDocs, options) {
```

Comment:
do we need to pass ratedDocs and bestDocs all around to mae this work?   Or externally could it just be "setofDocs" that gets set to one or the other?